### PR TITLE
Adding time series chart of ECCO and MEaSUREs sea surface height, including the mdx configuration snippet

### PR DESCRIPTION
**Pull request by dfelikson**

### CODE REVIEW

**Comment by dfelikson** [comment]
@danielfdsilva , @aboydnw , @hanbyul-here ... here's the initial pull request for the coastal discovery

**Comment by aboydnw** [comment]
Perfect, thanks @dfelikson !

To make this discovery operable, it'll need some metadata at the top. I usually just copy and paste from another discovery and change the values. You can use the discovery here as a template. Everything between the three dashes is what we need.

https://raw.githubusercontent.com/NASA-IMPACT/delta-config/develop/discoveries/agriculture-food-security-east-africa.discoveries.mdx

Here is an image with some more details.
image

**Comment by dfelikson** [comment]
I added the header but there's some placeholder stuff that we'll have to update (e.g., the cover image). @aboydnw, let me know if this is a valid header. And is there a way for us to be able to preview what this will look like in a browser?

**Comment by danielfdsilva** [comment]
:wave: @dfelikson
The properties of your file seem to be all fine, but the name of the file must be `[name].discoveries.mdx` otherwise it won't be picked up.
I enabled the deploy preview for this PR, so you'll be able to see your changes at https://deploy-preview-129--visex.netlify.app/

I'd also recommend that all your files have lowercase names, including the `.csv` one.

**Comment by dfelikson** [comment]
@danielfdsilva done! Changed the markdown filename to `[name].discoveries.mdx` and changed the csv to lowercase.

File: discoveries/coastal-flooding-and-slr.discoveries.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   <Prose>
+    ## Sea level rise off the coast of Bangladesh
+      This chart shows the increase in sea level off the coast of Bangladesh over time.
+      
+      [PLACEHOLDER -- DENIS WILL FILL IN WITH TEXT ABOUT THE CHART]
+
+  </Prose>
+  <Figure>
+    <Chart
+      dataPath={new URL('./ECCO_and_MEaSUREs_SSH.csv', import.meta.url).href}
```
Comment (danielfdsilva) [inline discoveries/coastal-flooding-and-slr.discoveries.mdx:27]: @dfelikson The dataPath here also needs updating

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

File: discoveries/coastal-flooding-and-slr.discoveries.mdx (hunk 1 above)
Comment (dfelikson) [reply discoveries/coastal-flooding-and-slr.discoveries.mdx]: Done!

File: discoveries/ecco_and_measures_ssh.csv [hunk 2]
```diff
@@ -0,0 +1,602 @@
+date, source, SSH (m)
```
Comment (danielfdsilva) [inline discoveries/ecco_and_measures_ssh.csv:1]: I just noticed that the column names have a leading space, which should not happen and is the reason why the chart does not show data.
```suggestion
date,source,SSH (m)
```

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

File: discoveries/ecco_and_measures_ssh.csv (hunk 2 above)
Comment (dfelikson) [reply discoveries/ecco_and_measures_ssh.csv]: Fixed - I removed white spaces but I left one in between "SSH" and "(m)". Let's see if that works ...

**Comment by danielfdsilva** [comment]
@dfelikson showing up nicely https://deploy-preview-129--visex.netlify.app/eis/discoveries/eis-coastal-risks

**Comment by dfelikson** [comment]
Excellent. Thanks for your help, @danielfdsilva! I'll probably have an update to this plot - there shouldn't be an offset between the data. Will work on that today ...

**Comment by dfelikson** [comment]
I added three more datasets as csv files and I put in configuration code for these into the markdown file. Please let me know if there are any errors or issues with these time series.

**Comment by danielfdsilva** [comment]
@dfelikson You stumbled upon a bug in the chart! 🐛

There is a problem with rendering data of a single dimension. I logged the [issue](https://github.com/NASA-IMPACT/delta-ui/issues/316) and we'll be fixing it. Have a look for a more detailed explanation.

For the time being you can work around the issue by adding an additional column called `type`, whose value is the same for every row (this is the label that will show up in the chart legend), and then set `idKey='type'` as a chart property.

The affected csv files are:
- groundwater_time_series.csv
- ecco_and_measures_ssh.csv
- coastal_land_loss_time_series.csv

**Comment by dfelikson** [comment]
@danielfdsilva - done! I added a `type` column and set `idKey='type'` for each of those three time series.

File: discoveries/coastal-flooding-and-slr.discoveries.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+      [PLACEHOLDER -- MORE TEXT COMING]
+
+  </Prose>
+  <Figure>
+    <Chart
+      dataPath={new URL('./coastal_land_loss_time_series.csv', import.meta.url).href}
+      dateFormat="%Y"
+      idKey='type'
+      xKey='date'
```
Comment (danielfdsilva) [inline discoveries/coastal-flooding-and-slr.discoveries.mdx:72]: It seems that this should be `year` instead of `date`
```suggestion
      xKey='year'
```

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

**Comment by danielfdsilva** [comment]
@dfelikson I went ahead and committed the suggestion. All the charts are showing up at https://deploy-preview-129--visex.netlify.app/eis/discoveries/eis-coastal-risks

**Comment by dfelikson** [comment]
Thanks, @danielfdsilva. Things look good! We'll make some more tweaks and I'll keep pushing to the Git repo.

**Pull request merged**